cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -31,4 +31,23 @@ describe('ui-inventory: keyboard shortcut', () => {
     InventoryMainButton.waitModalLoading('Keyboard shortcuts');
     InventoryMainButton.verifyInventoryDropdownIsShown('false');
   });
+
+  it('C345298 Keyboard Shortcut. Test the functionality of the different shortcut keys', { tags: [testTypes.smoke] }, () => {
+    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+
+    InventoryMainButton.openInventoryMenu();
+    InventoryMainButton.openShortcuts();
+
+    InventoryMainButton.waitModalLoading('Keyboard shortcuts');
+    InventoryMainButton.createNewRecordByShortcuts();
+    InventoryMainButton.closeShortcutsViaUi();
+    InventoryMainButton.fillInstanceInfoViaUi();
```

Comment:
from TC:
"Enter a brief instance record.
And when done, then click "Save and close""
I suggest to rewtite to 2 methods - fillRequiredFields + SaveAndClose.

Reply:
I Want to check Shortcut CTRL+S